app/views/incomes/confirm_save_Income.scala.html

```diff
@@ -48,7 +48,7 @@
     <div class="grid-row">
         <div class="grid grid-2-3">
             <div class="inner-block">
-                <h2 class="heading-medium heading--underlined flush--top">@Messages("tai.taxYear",s"${Dates.formatDate(TaxYearResolver.startOfCurrentTaxYear)}",s"${Dates.formatDate(TaxYearResolver.endOfCurrentTaxYear)}")</h2>
+                <h2 class="flush--top">@Messages("tai.taxYear",s"${Dates.formatDate(TaxYearResolver.startOfCurrentTaxYear)}",s"${Dates.formatDate(TaxYearResolver.endOfCurrentTaxYear)}")</h2>
```

Comment:
More a discussion on this page which existed previously, but what do you think about the hardcoded things like these dates in the views? Should these dates be passed in with the view model?

Reply:
Its probably fine when its static stuff like this but I don't know if the view should know about TaxYearResolver

Reply:
I was thinking along the same lines. The date is something that could easily be passed in